lib/ideas/bloc/ideas_bloc.dart

```diff
@@ -10,14 +10,14 @@ class IdeasBloc extends Bloc<IdeasEvent, IdeasState> {
   IdeasBloc(
     this._recipesRepository,
   ) : super(IdeasLoading()) {
-    on<IdeasLoaded>(_onIdeasLoaded);
+    on<LoadIdeas>(_onLoadIdeas);
```

Comment:
In VGV we try to follow the same conventions across projects.  Regarding bloc event naming it should be 
`BlocSubject + Noun (optional) + Verb (event)`
In this case:
`Ideas + (nothing in this case) + Loaded`

You can read more about this in here:
https://bloclibrary.dev/#/blocnamingconventions?id=event-conventions